agent/stats/queue_test.go

```diff
@@ -37,32 +36,33 @@ const (
 	predictableInt64Overflow = math.MaxInt64 / int64(2)
 )
 
+var now time.Time = time.Now()
+
 func getTimestamps() []time.Time {
 	return []time.Time{
-		parseNanoTime("2015-02-12T21:22:05.131117533Z"),
```

Comment:
is this change necessary now that you don't rely on relative timestamp but on sent flag?

Reply:
it's not, but i thought it seemed more readable this way rather than having to grok the time difference between stats based on this 9-digit decimal.